blog/static/css/style.css

```
@@ -0,0 +1,142 @@
+:root {
+    --bg: #ffffff;
+    --fg: #1b1f24;
+    --muted: #5b6570;
+    --accent: #b5179e;
+    --accent-2: #3a0ca3;
+    --border: #e6e8eb;
+    --code-bg: #2b303b;
+    --max: 46rem;
+    --font: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif;
```

Stylelint (17.13.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)